web/v1/redshift/sql-runner/sql/custom/01-indicative-export/04-indicative-export-upsert.sql

```diff
@@ -0,0 +1,11 @@
+-- 3. DELETE - INSERT to production
+
+BEGIN; --it is safest to use a transaction, in case of failure.
+
+  DELETE FROM {{.output_schema}}.indicative_export{{.entropy}}
+    WHERE event_id IN (SELECT event_id FROM {{.scratch_schema}}.indicative_export_staged{{.entropy}});
```

Comment:
I would add table scan limit [as mention](https://github.com/snowplow-incubator/indicative-data-model/pull/1#discussion_r656123044). Your call though. Not the end of the world